pkgs/tools/virtualization/multipass/default.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  qtbase,
+  wrapQtAppsHook,
+}: let
+  pname = "multipass";
+  version = "1.11.0";
+in
+  stdenv.mkDerivation {
+    name = "${pname}-${version}";
```

`name` was deprecated years ago. Just inherit them:


```suggestion
    inherit pname version;
```